{
  "exploitdb": [],
  "itw": [
    {
      "id": "CVE-2021-29562",
      "description": "TensorFlow is an end-to-end open source platform for machine learning. An attacker can cause a denial of service by exploiting a `CHECK`-failure coming from the implementation of `tf.raw_ops.IRFFT`. The fix will be included in TensorFlow 2.5.0. We will also cherrypick this commit on TensorFlow 2.4.2, TensorFlow 2.3.3, TensorFlow 2.2.3 and TensorFlow 2.1.4, as these are also affected and still in supported range.",
      "referenceURL": "https://nvd.nist.gov/vuln/detail/CVE-2021-29562",
      "reportURL": "https://github.com/tensorflow/tensorflow/security/advisories/GHSA-36vm-xw34-x4pj",
      "timeStamp": "2021-05-18T20:07:00Z"
    }
  ],
  "trickest": [],
  "nomisec": [],
  "nuclei": [],
  "metasploit": [],
  "holloways": [],
  "openpoc": [
    {
      "cve": "CVE-2021-29562",
      "url": "https://github.com/tensorflow/tensorflow/security/advisories/GHSA-36vm-xw34-x4pj",
      "added_at": "2021-05-18T20:07:00Z",
      "score": 0.4,
      "template_for": ""
    }
  ]
}
